src/utils/tekster.ts

```diff
@@ -0,0 +1,9 @@
+export const hentPeriodeTittelMedTall = (
+  perioder: any[],
+  periodenr: number,
```

Comment:
rename til `oppholdsnr` for å være konsistent

Reply:
done :)